src/Illuminate/Container/Container.php

```diff
@@ -24,112 +24,112 @@ class Container implements ArrayAccess, ContainerContract
     /**
      * An array of the types that have been resolved.
      *
-     * @var array
+     * @var bool[string]
```

Comment:
`array[string]`?

Reply:
Isn't an array of strings `string[]`?

Reply:
Wth does bool[string] mean?

Reply:
this syntax means that it is an associative array
keys are strings.
values are boolean.

Reply:
string[]  syntax means that it is an sequential array
keys are numbers.
values are string. 
`['a', 'b'];`

Reply:
This syntax is not used in the framework because it is not a well-accepted standard and has poor tool support, with some software unable to parse it and crashing.